src/Organic/AdsConfig.php

```diff
@@ -52,7 +52,14 @@ public function __construct( array $rawAdsConfig, array $rawAdsRefreshRates ) {
         $this->prebidUrl = $prebid['useBuild'] ?? $this->fallbackUrl;
     }
 
-    public function getPrebidBuildUrl() : string {
-        return $this->prebidUrl;
+    public function getPrebidBuildUrl() : array {
+        $moduleSrc = '';
+        if ( preg_match( $this->MODERN_PREBID_RE, $this->prebidUrl ) ) {
+            $moduleSrc = str_replace( '.js', '.m.js', $this->prebidUrl );
+        };
+        return [
+            'default' => $this->prebidUrl,
+            'module' => $moduleSrc,
+        ];
```

Comment:
Similar comment here to [this one](https://github.com/orgnc/wordpress-plugin/pull/168/files#r1180504311)